raiden/tests/utils/events.py

```diff
@@ -127,6 +128,20 @@ def must_have_events(event_list: List[TM], *args) -> bool:
     return True
 
 
+def has_event_of_types(events: List[Event], event_types: List[Type]) -> bool:
+    for event in events:
+        for event_type in event_types:
```

Comment:
[`isinstance`](https://docs.python.org/3/library/functions.html#isinstance) accepts a tuple as an argument, please use that instead.

Reply:
this is missing

Reply:
The change from `List[Type]`  to `Tuple[Type, ...]`? Added it.

Reply:
```diff
-         for event_type in event_types: 
-            if isinstance(event, event_type):
-                return True 
+         if isinstance(event, event_types):
+            return True 
```